### PR TITLE
first commit - html and css built

### PR DESCRIPTION
**Pull request by Sergei-Nikolaienko**
[DEMO LINK] https://Sergei-Nikolaienko.github.io/layout_html-form/
[TEST REPORT LINK] https://Sergei-Nikolaienko.github.io/layout_html-form/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,17 @@
 /* styles go here */
+
+.form__section {
+  border: 1px solid black;
```
Comment (alexandra-protyanova) [inline src/style.css:4]: your tests are not passing because of this borders, use `legend` tag instead

File: src/index.html [hunk 2]
```diff
@@ -1,17 +1,194 @@
...
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <!-- Personal Info Section -->
+      <fieldset class="form__section">
+        <span class="headline">Personal information</span>
```
Comment (alexandra-protyanova) [inline src/index.html:19]: in this task divs and spans are absolutely redundant

File: src/index.html [hunk 3]
```diff
@@ -1,17 +1,194 @@
...
+            >
+            Yes
+          </label>
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="hatesCats"
+            >
+            No
```
Comment (alexandra-protyanova) [inline src/index.html:114]:
```suggestion
              No
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,17 @@
...
+  margin-bottom: 20px;
+}
+
+.form__field {
+  margin-bottom: 10px;
+}
+
+.headline {
+  position: relative;
```
Comment (alexandra-protyanova) [inline src/style.css:13]: you only need margins and `display: block` in this task in styles

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 5]
```diff
@@ -1,17 +1,174 @@
...
+      <fieldset class="form__section">
+        <legend>Personal information</legend>
+
+        <label class="form__field" for="surname">Surname:
+          <input 
+            type="text" 
+            id="surname" 
+            autocomplete="off"
+          >
+        </label>
```
Comment (SanyaBratashchuk) [inline src/index.html:27]:
```suggestion
        <label class="form__field" for="surname">
          Surname:
          <input 
            type="text" 
            id="surname" 
            autocomplete="off"
          >
        </label>
```

File: src/index.html [hunk 6]
```diff
@@ -1,17 +1,174 @@
...
+
+        <label class="form__field" for="name">Name:
```
Comment (SanyaBratashchuk) [inline src/index.html:29]:
```suggestion
        <label class="form__field" for="name">
          Name:
```

Comment (SanyaBratashchuk) [reply src/index.html:29]: Fix it everywhere

File: src/index.html [hunk 7]
```diff
@@ -1,17 +1,174 @@
...
+            name="cats" 
+            value="lovesCats"
+          >
+            Yes
+          <input 
+          type="radio" 
+          name="cats" 
+          value="hatesCats"
+          >
+            No
```
Comment (SanyaBratashchuk) [inline src/index.html:103]: Wrap each radio with label to make `yes` and `no` words clickable

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 8]
```diff
@@ -1,17 +1,191 @@
...
+        <legend class="headline">An interesting fact about you!</legend>
+        <div class="cat-question">
+          <label class="cat-yes">  
+            Do you love cats?
+            <input 
+              type="radio" 
+              name="cats" 
+              value="lovesCats"
+            >
+              Yes
```
Comment (SanyaBratashchuk) [inline src/index.html:106]:
```suggestion
            <input 
              type="radio" 
              name="cats" 
              value="lovesCats"
            >
            Yes
```

File: src/index.html [hunk 9]
```diff
@@ -1,17 +1,191 @@
...
+          </label>
+          <label class="cat-no">
+            <input 
+            type="radio" 
+            name="cats" 
+            value="hatesCats"
+            >
+              No
```
Comment (SanyaBratashchuk) [inline src/index.html:114]:
```suggestion
          <label>
            <input 
            type="radio" 
            name="cats" 
            value="hatesCats"
            >
            No
```

File: src/index.html [hunk 10]
```diff
@@ -1,17 +1,191 @@
...
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+
+      <!-- User Facts Section -->
+      <fieldset class="form__section">
+        <legend class="headline">An interesting fact about you!</legend>
+        <div class="cat-question">
+          <label class="cat-yes">
```
Comment (SanyaBratashchuk) [inline src/index.html:99]:
```suggestion
          <label>  
```

File: src/style.css [hunk 11]
```diff
@@ -1 +1,18 @@
...
+  margin-bottom: 10px;
+}
+
+.form__section {
+  margin-bottom: 20px;
+}
+
+.form__section > .form__field:last-of-type {
+  margin-bottom: 0;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:18]:
```suggestion
.form__field:last-child {
  margin-bottom: 0;
}
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 12]
```diff
@@ -1 +1,18 @@
 /* styles go here */
+
+.form__field {
```
Comment (DarkMistyRoom) [inline src/style.css:3]:
```suggestion
.form__field {
```

File: src/index.html [hunk 13]
```diff
@@ -1,17 +1,191 @@
...
+          >
+        </label>
+      </fieldset>
+
+      <!-- User Facts Section -->
+      <fieldset class="form__section">
+        <legend class="headline">An interesting fact about you!</legend>
+        <div class="cat-question">
+          <label>  
+            Do you love cats?
```
Comment (DarkMistyRoom) [inline src/index.html:100]: Move _Do you love cats?_ out the label because when you click on the question, yes becomes selected

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
remove spaces from textarea
image

**Comment by Sergei-Nikolaienko** [comment]
the only fix I've found for text area is by placing the closing tag on the same line as the opening one, however due to this the tests won't pass

**Review by DarkMistyRoom — APPROVED** [review]

**Pull request closed without merging**